Sts1CobcSw/Hal/Spi.hpp

```diff
@@ -10,6 +10,25 @@
 
 namespace sts1cobcsw::hal
 {
+class Spi
+{
+private:
+    RODOS::HAL_SPI spi;
+    RODOS::CommBuffer<std::int64_t> transferEnd;
+
+public:
+    friend auto Initialize(Spi * spiClass, std::int32_t baudRate) -> void;
+
+    template<typename T, std::size_t extent>
+    friend auto ReadFrom(Spi * spiClass, std::span<T const, extent> data, std::int64_t timeout)
+        -> void;
+
+    template<typename T, std::size_t extent>
+    friend auto WriteTo(Spi * spiClass, std::span<T, extent> data, std::int64_t timeout) -> void;
```

Comment:
Writing to the SPI does not require a span with mutable elements, so make the `T` `const`.